lib/CL/devices/pthread/pthread_scheduler.c

```diff
@@ -28,6 +28,7 @@
 #endif
 
 #include <pthread.h>
+#include <stdatomic.h>
```

Comment:
We already have macros (`POCL_ATOMIC_INC` and so on) for atomics in `pocl_cl.h`, it would be great not to make new code less portable..

Reply:
I didn't know about them. Switched now to POCL_ATOMIC_INC.